<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Menu">
       <widget class="QPushButton" name="log_out">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
       <widget class="QListWidget" name="conectedUsers">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>271</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>chat</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Chat">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>550</y>
          <width>531</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Message:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="lnMessage"/>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="btnSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="btnSendFile">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Desitnation:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="cmbDestination">
           <item>
            <property name="text">
             <string>Server</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="lstMessages">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>531</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="backmenu">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>531</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lnClientName">
           <property name="placeholderText">
            <string>Enter Client Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbStatus">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Available</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Away</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Busy</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Login">
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>240</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>User</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>340</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Log in</string>
        </property>
       </widget>
       <widget class="QPushButton" name="singin">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>370</y>
          <width>91</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Sign in</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Sign">
       <widget class="QLineEdit" name="Name">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>200</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="signin">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>370</y>
          <width>101</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sign In</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>240</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>User </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sign In</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>200</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="userdate">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>320</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>280</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>320</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Username">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>240</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="loginb">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>410</y>
          <width>101</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Log In</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="password">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>280</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExot"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionExot">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
